cli/types/cypress.d.ts

```diff
@@ -3432,6 +3432,12 @@ declare namespace Cypress {
     specPattern?: '**/*.cy.ts'
   }
 
+  type ErroredFramework = { 
```

Comment:
I don't think this belongs in `cli` - it's not used here. What about putting it closer to where it is used?